### PR TITLE
Create CarController.java

### PR DESCRIPTION
**Pull request by markgrebe**

### CODE REVIEW

File: CarController.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("cars")
+public class CarController {
+
+    @Autowired
+    private CarDao carDao;
```
Comment (mgrebe-cu) [inline CarController.java:17]: Should this really be private?

File: CarController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public Car getOne(@PathVariable int id){
+        Optional<Car> optionalCar = carDao.findById(id);
+        return optionalCar.isPresent() ? optionalCar.get() : null;
+    }
+
+    @GetMapping("/make/{make}")
+    public List<Car> getMake(@PathVariable String make){
+        return carDao.findByMake(make);
+    }
+
```
Comment (mgrebe-cu) [inline CarController.java:42]: Is this method really needed?

**Review by mgrebe-cu — CHANGES_REQUESTED** [review]

File: CarController.java (hunk 1 above)
Comment (markgrebe) [reply CarController.java:17]: Your right it should be protected instead.

File: CarController.java (hunk 2 above)
Comment (markgrebe) [reply CarController.java:42]: Yes, it is !  See paragrapsh 4.3.5.6 of spec.

**Pull request merged**